recipe/meta.yaml

```diff
@@ -1,41 +1,44 @@
 {% set name = "anndata" %}
-{% set version = "0.7.8" %}
+{% set version = "0.8.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1efd7eb40839e0325bb066238280228a980d7dde6410793dbff2835f44a2d3ef
+  sha256: 94d2cc6f76c0317c0ac28564e3092b313b7ad19c737d66701961f3e620b9066e
 
 build:
   number: 1
   script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py<35]
+  noarch: python 
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.6
     - pip
     - flit-core
     - setuptools_scm
     - pytoml
   run:
-    - python
+    - python >=3.6 
     - pandas >=0.23.0
     - scipy >=1.4
     - h5py
     - natsort
-    - importlib_metadata  # [py<38]
-    - packaging
+    - importlib_metadata
+    - typing_extensions
```

Comment:
Both of those are only necessary in Python <3.8.

Can you make it so they don’t get installed in Python 3.8+?